node_modules/gh-orgunit/lib/api.js

```diff
@@ -530,10 +530,11 @@ var getOrgUnitSeries = module.exports.getOrgUnitSeries = function(ctx, id, limit
  * @param  {Context}        ctx                 Standard context containing the current user and the current app
  * @param  {Number}         id                  The id of the organisational unit to add the serie(s) to
  * @param  {Number[]}       series              The id(s) of the serie(s) to add to the organisational unit
+ * @param  {Number}         [fromOrgUnitId]     The id of the organisational unit the series are borrowed from
```

Comment:
This seems confusing, as you can add multiple event series to an organisational unit at once, but you pass in a single "borrowed from" id. I think we either need to reduce the scope of this function to adding 1 series per org unit or specifying the series id + borrowed from together for each series.